src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/gtfs/TripTestsIT.java

```diff
@@ -0,0 +1,138 @@
+    
+    response
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(7))
+      .body("id[1]", notNullValue())
+      .body("routeId[1]", notNullValue())
+      .body("scheduleId[1]", notNullValue())
+      .body("headsign[1]", is("Mikkeli matkakeskus"));
+    
+    String routeId = response.body().jsonPath().getString("routeId[1]");
+    String ScheduleId = response.body().jsonPath().getString("scheduleId[1]");
+    
+    assertFound(String.format("/organizations/%s/transportRoutes/%s", organizationId, routeId));
+    assertFound(String.format("/organizations/%s/transportSchedules/%s", organizationId, ScheduleId));
+    
+  }
+  
+  @Test
+  public void testFindTrip() {
+    String organizationId = getOrganizationId(0);
+    Response response = given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportTrips/{tripId}", organizationId, getOrganizationTripId(organizationId, 2));
+    
+    response
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("routeId", notNullValue())
+      .body("scheduleId", notNullValue())
+      .body("headsign", is("Pajulankylä"));
+    
+    String routeId = response.body().jsonPath().getString("routeId");
+    String ScheduleId = response.body().jsonPath().getString("scheduleId");
+    
+    assertFound(String.format("/organizations/%s/transportRoutes/%s", organizationId, routeId));
+    assertFound(String.format("/organizations/%s/transportSchedules/%s", organizationId, ScheduleId));
+  }
+  
+  @Test
+  public void testOrganizationTripsNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationTripId = getOrganizationTripId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/transportTrips/%s", organizationId, organizationTripId));
+    assertEquals(7, countApiList(String.format("/organizations/%s/transportTrips", organizationId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/transportTrips/%s", organizationId, malformedId));
+    }
+    
+    assertNotFound(String.format("/organizations/%s/transportTrips/%s", incorrectOrganizationId, organizationTripId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/transportTrips", incorrectOrganizationId)));
+  }
+
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+  
+  private void createGtfsSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createGtfsSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)